nvmw.bat

```diff
@@ -75,27 +82,61 @@ exit /b 0
 :install
 setlocal
 
+set NODE_TYPE=node
 set NODE_VERSION=%1
-if not %NODE_VERSION:~0,1% == v (
-  set NODE_VERSION=v%1
+
+:: nvmw install iojs-v1.0.2
+if "%NODE_VERSION:~4,1%" == "-" (
+  for /f "tokens=1,2,* delims=-" %%a in ("%NODE_VERSION%") do (
+    set NODE_TYPE=%%a
+    set NODE_VERSION=%%b
+  )
 )
 
-if %OS_ARCH% == 32 (
-  set NODE_EXE_URL=%NVMW_NODEJS_ORG_MIRROR%/%NODE_VERSION%/node.exe
-) else (
-  set NODE_EXE_URL=%NVMW_NODEJS_ORG_MIRROR%/%NODE_VERSION%/x64/node.exe
+:: nvmw install iojs
+if %NODE_VERSION% == iojs (
+  set NODE_TYPE=iojs
+  set NODE_VERSION=latest
 )
 
-echo Start installing Node %NODE_VERSION% (x%OS_ARCH%)
+:: nvmw install node
+if %NODE_VERSION% == node (
+  set NODE_TYPE=node
+  set NODE_VERSION=latest
+)
+
+:: iojs-1.0.0, iojs-latest
+if not %NODE_VERSION:~0,1% == v if not %NODE_VERSION:~0,1% == l (
+  set NODE_VERSION=v%NODE_VERSION%
+)
+
+if %NODE_TYPE% == iojs (
+  set DIST_URL=%%
+  if %OS_ARCH% == 32 (
+    set NODE_EXE_URL=%NVMW_IOJS_ORG_MIRROR%/%NODE_VERSION%/win-x86/iojs.exe
+  ) else (
+    set NODE_EXE_URL=%NVMW_IOJS_ORG_MIRROR%/%NODE_VERSION%/win-x64/iojs.exe
+  )
+) else (
+  if %OS_ARCH% == 32 (
+    set NODE_EXE_URL=%NVMW_NODEJS_ORG_MIRROR%/%NODE_VERSION%/node.exe
+  ) else (
+    set NODE_EXE_URL=%NVMW_NODEJS_ORG_MIRROR%/%NODE_VERSION%/x64/node.exe
+  )
+)
 
 set "NODE_HOME=%NVMW_HOME%%NODE_VERSION%"
+if %NODE_TYPE% == iojs (
+  set "NODE_HOME=%NVMW_HOME%%NODE_TYPE%\%NODE_VERSION%"

```

Comment:
@hakobera I change to save node version as before to compatibility now. If there no other problems, this pr can merge  now.